modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/security/other/SecuritySlf4jTestCase.java

```diff
@@ -22,7 +22,7 @@
 import org.junit.Test;
 import org.slf4j.ext.EventData;
 
-@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/security/src/main/java"})//,"modules/testcases/jdkAll/Eap71x-Proposed/security/src/main/java","modules/testcases/jdkAll/Eap71x/security/src/main/java"})
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/security/src/main/java","modules/testcases/jdkAll/Eap7/security/src/main/java","modules/testcases/jdkAll/Eap71x/security/src/main/java"})
```

Comment:
Could you remove "modules/testcases/jdkAll/Eap7/security/src/main/java", ? The fix is not included there (at that branch).

Reply:
Ah I thought the fix would be there. Sorry, removed.

Reply:
SecuritySlf4jTestCase has passed on Jenkins. Thank you.